.classpath

```diff
@@ -5,6 +5,10 @@
 	<classpathentry kind="var" path="networktables" sourcepath="networktables.sources"/>
 	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-1.8"/>
 	<classpathentry kind="var" path="USERLIBS_DIR/navx_frc.jar" sourcepath="USERLIBS_DIR/navx_frc-sources.jar"/>
-	<classpathentry kind="var" path="USERLIBS_DIR/CTRLib.jar"/>
+        <classpathentry kind="var" path="USERLIBS_DIR/CTRLib.jar" sourcepath="USERLIBS_DIR/CTRLib-javadoc.jar"/>
```

Comment:
I was annoyed by the lack of CTRlib javadoc too and started adding it, but right now it's not in 4904-USERLIBS so we can't really include it here. Also a couple lines down the NavX javadoc was readded in spite of already being on line 7.